prow/getting_started_deploy.md

```diff
@@ -202,49 +215,20 @@ to start receiving GitHub events!
 
 ## Add the webhook to GitHub
```

Comment:
Suggest change this to something like "Setup webhook for prow user"

Reply:
But this section isn't needed when using a GH app

Reply:
The new line 218 you added does mention set up webhook though

Reply:
Well, it is needed once for the entire app and will be used everywhere it is installed. I guess the larger question behind this is if we should keep existing PAT-related docs.

Reply:
Agreed on the larger question. My initial thought was actually we want to keep it, but not very sure. 

However, if we do want to keep it, we probably should make it very clear that gh app is the recommended approach

Reply:
Hm I tend to say to not keep it because existing users already know this and new users should be setting up an app. Furthermore we'd then also have to have another set of `getting_started_{s3,gcs},yaml` manifests that use PAT. WDYT?

Reply:
Makes sense to me. I personally don't use this doc that much as operator of a couple of prow instances, so yeah I agree that existing user already know what to do. Also the set of started yaml is a great point.